MekHQ/data/universe/news.xml

```diff
@@ -711,4 +711,17 @@ Unfortunately for Victor Steiner-Davion, he returns home to find himself an Arch
 		<service>INN</service>
 		<location>Outreach</location>
 	</newsItem>
-</news>
\ No newline at end of file
+	<newsItem>
```

Comment:
This is a typical newsItem element. Later request can include batches of several newsItems at a time to avoid spamming up the queue.